.gitignore

```diff
@@ -52,3 +52,6 @@ config.stamp
 Session.vim
 .cargo
 no_llvm_build
+# Generated when dumping Graphviz output for debugging:
+/mir_dump/
+/*.dot
```

Comment:
I think this should be reverted. In fact I just removed these not too long ago.^^

I do sometimes create `mir_dump` folders during debugging, and those are junk that I want to clean up. They should *not* be automatically ignored by git. If you are okay with junk in your working dir, that's okay -- git has `.git/info/exclude` for you to locally configure stuff you want ignored. But please don't interfere with other people who want to keep their working dir clean.

Reply:
In particular see the comment at the top of this file:
```
# This file should only ignore things that are generated during a build,
# generated by common IDEs, and optional files controlled by the user
# that affect the build (such as config.toml).
```